opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -713,6 +714,30 @@ std::vector<std::array<int,2>> CpGrid::mapLeafIndexSetToLocalCartesianIndexSets(
     return leafIdx_to_localCartesianIdxSets;
 }
 
+std::vector<std::array<int,3>> CpGrid::localIJK(const std::string& lgr_name) const
```

Comment:
Should we make this a free function instead of a member function?

Reply:
Yes, we should. Then lgr_names_ could be replace by a call of getLgrNameToLevel()